src/main/resources/templates/layouts/baseLayout.html

```diff
@@ -29,5 +29,10 @@
         </main>
     </div>
     <div th:replace="fragments/footer :: footer"></div>
+    <script type="text/javascript">
```

Comment:
The empty body class requiring this JS block could get further separated from this block as more features are added to this layout HTML page. This could be a pain for a developer to figure out.

Also, our JS is in JS files in the CDN repository, we should have a consistent approach.

Reply:
Sorry, lifted from abridged. Will move this to cdn.

Reply:
It's also worth considering using jQuery to append the class in this case which would be more readable: `$('body').addClass('js-enabled');`

Reply:
This has been done, so this PR is now dependent on https://github.com/companieshouse/cdn.ch.gov.uk/pull/261